app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html

```diff
@@ -53,22 +60,22 @@ <h2>@messages("nonTaxableTrustRegistrationAccessCode.subHeading")</h2>
 
         <p>@messages("nonTaxableTrustRegistrationAccessCode.p5")</p>
 
-        @components.warning("nonTaxableTrustRegistrationAccessCode")
+        @warning("nonTaxableTrustRegistrationAccessCode")
 
         <h2>@messages("nonTaxableTrustRegistrationAccessCode.subHeading2")</h2>
 
         <p>@messages("nonTaxableTrustRegistrationAccessCode.p6")</p>
 
         <div class="panel-indent">
```

Comment:
Somehow missed this old `panel-indent` class when replacing all for `govuk-inset-text`

Reply:
Fixed now, and I've added the missing message key for `whenTrustSetupDate.checkYourAnswersLabel`